charts/marketplace/gcp/release.sh

```diff
@@ -46,10 +46,10 @@ function retag() {
 }
 
 # Ensure chart app version matches schema.yaml version
-sed -i .bak "s/version: .*/version: ${target_tag}/" values.yaml
+sed "-i .bak" "s/version: .*/version: ${target_tag}/" values.yaml
```

Comment:
This would create backup file `values.yaml .bak`, note the space before the `.bak` suffix

```suggestion
sed -i.bak "s/version: .*/version: ${target_tag}/" values.yaml
```